packages/stage-ui/src/styles/derived-utilities.css

```
@@ -151,3 +165,15 @@
 @utility bg-derived-darker {
   background-color: oklch(from var(--cm-bg-color, var(--color-border-subtle)) calc(l - 0.09) calc(c - 0.014) h);
 }
+
+@utility bg-shortcut-solid-derived {
```

Stylelint (17.11.1)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)